RiotSharp/Endpoints/Interfaces/ITftMatchEndpoint.cs

```diff
@@ -0,0 +1,28 @@
+﻿using RiotSharp.Endpoints.TftMatchEndpoint;
+using RiotSharp.Misc;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RiotSharp.Endpoints.Interfaces
+{
+    public interface ITftMatchEndpoint
+    {
+        /// <summary>
+        /// Gets a list of match ids by puuid
+        /// </summary>
+        /// <param name="region">Region in which the summoner is.</param>
+        /// <param name="puuid"></param>
+        /// <returns>A list of strings</returns>
+        Task<List<string>> GetTftMatchIdsByPuuidAsync(Region region, string puuid, int count = 20);
```

Comment:
Can you remove `ByPuuid` from the naming to match the naming of the match endpoint.